.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,40 @@
+---
+name: Bug report
+about: Found a bug? Tell us and help us improve
+title: ''
+labels: a:bug
+assignees: ''
+
+---
+
+**Does the issue reproduce with the latest available release? (y/n)**
+
+**Your gotify server is running on**
```

Comment:
Could this be phrased as a question, then it is more consistent with the other headlines.